pkg/util/enc.go

```diff
@@ -55,3 +55,21 @@ func TryToDecodeBase64Gzip(data []byte) (string, error) {
 
 	return string(plainText), nil
 }
+
+// EncodeGzipBase64 encodes the provided string to base64 and then gzips it.
```

Comment:
Comment: `EncodeGzipBase64 compressed the provided string then encode it to a base64 string`